src/main/java/io/github/nucleuspowered/nucleus/internal/KitInventory.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * This file is part of Nucleus, licensed under the MIT License (MIT). See the LICENSE.txt file
+ * at the root of this project for more details.
+ */
+package io.github.nucleuspowered.nucleus.internal;
+
+import com.google.common.collect.Lists;
+import ninja.leaping.configurate.objectmapping.Setting;
+import ninja.leaping.configurate.objectmapping.serialize.ConfigSerializable;
+import org.spongepowered.api.item.inventory.ItemStack;
+
+import java.util.List;
+
+@ConfigSerializable
+public class KitInventory {
```

Comment:
@dualspiral Look again, I think this is better. It gives server admins the ability to specify the units AND the actual interval